py/selenium/webdriver/ie/webdriver.py

```diff
@@ -79,7 +84,7 @@ def __init__(self, executable_path='IEDriverServer.exe', capabilities=None,
         self.iedriver.start()
 
         if capabilities is None:
-            capabilities = DesiredCapabilities.INTERNETEXPLORER
+            capabilities = DesiredCapabilities.INTERNETEXPLORER.copy()
```

Comment:
I changed this line, but based on the overall logic, this line is never ran....

Reply:
Yeah looks like it, can probably be removed.